client/jetpack-cloud/sections/partner-portal/license-lightbox/index.tsx

```diff
@@ -20,6 +20,7 @@ export type LicenseLightBoxProps = {
 	onActivate: ( product: APIProductFamilyProduct ) => void;
 	onClose: () => void;
 	product: APIProductFamilyProduct;
+	extraContent?: JSX.Element;
```

Comment:
To avoid confusion that we are adding extra content in the sidebar and not in the main container, maybe we can name this `extraAsideContent` and make this more specific to the aside content. This way, the intention is more apparent.

Reply:
Makes sense!